test/integration/main_test.go

```diff
@@ -169,19 +184,24 @@ func restartInsightsOperator(t *testing.T) {
 	t.Log(errPod)
 }
 
-func checkPodsLogs(t *testing.T, kubeClient *kubernetes.Clientset, message string) {
+func _checkPodsLogs(t *testing.T, kubeClient *kubernetes.Clientset, message string, newLogs bool) {
```

Comment:
I'm not sure how underscore function name is used in go, @martinkunc ?

Reply:
You are right, go devs are a bit stricter about naming. Please use camelCase for private and PascalCase for public methods. Underscores should be only in file names, not in variables. See here: https://golang.org/doc/effective_go.html#mixed-caps

Reply:
By this naming, it looks like most of the main_test.go functions should also be renamed - most functions are in camelCase but are used as public ones

Reply:
Ok. And regarding the public functions. Public in Go means that they are accessible from other packages, which is also making a contract with other packages that they will keep working, which is why we tend not to use them if we don't have to.
In case of tests, I think we are in same packge, so we don't need to expose functions for someone else, except for Test... functions, executed by test runner.